includes/class-wporg-gp-translation-events-translation-listener.php

```diff
@@ -114,11 +114,13 @@ private function get_active_events( DateTime $at ): array {
 	 * @return WP_Post[]
 	 */
 	private function select_events_user_is_registered_for( array $events, int $user_id ): array {
+		$attending_event_ids = get_user_meta( $user_id, WPORG_GP_Translation_Events_Route::USER_META_KEY_ATTENDING, true ) ?? [];
+		$attending_event_ids = array_keys( $attending_event_ids );
+
 		return array_filter(
 			$events,
-			function ( $event ) {
-				// TODO.
-				return true;
+			function ( $event ) use ( $attending_event_ids ) {
+				return array_key_exists( $event->ID, $attending_event_ids );
```

Comment:
I personally prefer this since it's visually clearer to me.
```suggestion
				return isset( $attending_event_ids[ $event->ID ] );
```